### ISSUE TITLE
I cant Run line 3/2

### ISSUE THREAD

**Issue by 12tae12** [nerdaxic/glados-voice-assistant#32]
Can't run
cp ~/glados-voice-assistant/settings.env.sample ~/glados-voice-assistant/settings.env && nano ~/glados-voice-assistant/settings.env
Need help

**Comment by eternalliving** [comment]
What error message are you getting? Is it saying the file isn't found, or access denied? You could also try replacing nano with either vi or vim or other txt editor of your choice.

**Comment by 12tae12** [comment]
It's saying the file isn't found

**Comment by eternalliving** [comment]
Sorry, I just looked through the instructions you are following and see they are outdated. It no longer uses a settings.env file but rather a yaml file under the settings folder. 

I’ll work on updating the instructions. For now, edit the glados_settings.yaml file and create a new home_assistant_settings.yaml file and edit to your needs.

**Comment by 12tae12** [comment]
Ok

**Comment by 12tae12** [comment]
I tried to edit and I don't know enough python to edit it to my needs.

**Comment by eternalliving** [comment]
Are you trying to use this repo for home assistant? Both of the settings files are in yaml and there is instructions in them on configuring it to your needs. 

Let me know where your getting stuck and I can help walk you through it.

**Comment by 12tae12** [comment]
No. I mean I know almost no python at all and don't know how to run it with this.

**Comment by 12tae12** [comment]
I just started learning...
